src/resources/js/ticket-details.js

```diff
@@ -26,35 +31,31 @@ var tribe_ticket_details = tribe_ticket_details || {};
 			return;
 		}
 
-		var parent          = trigger.closest( '.tribe-block__tickets__item__details__summary' );
-		var target_selector = trigger.getAttribute( 'aria-controls' );
-		var target          = document.getElementById( target_selector );
+		var $parent = $( trigger.closest( '.tribe-block__tickets__item__details__summary' ) );
```

Comment:
the `closest` api is not available for IE11. if you use jQuery's `closest` function (like `$( trigger ).closest()`), it'll be compatible with older browsers.